mlflow/types/schema.py

```diff
@@ -2,34 +2,50 @@
 from enum import Enum
 
 import numpy as np
+import pandas as pd
 from typing import Dict, Any, List, Union, Optional
 
+from pandas.core.dtypes.dtypes import PandasExtensionDtype
+
 from mlflow.exceptions import MlflowException
 
 
+def _pandas_string_type():
```

Comment:
What's going on here?

Reply:
String extention type is only available for pandas >= 1.0 and is recommended over object. We will just use object for pandas < 1.0.